yoti-sdk-api/src/main/java/com/yoti/api/client/Receipt.java

```diff
@@ -56,6 +60,14 @@ public ExtraData getExtraData() {
         return extraData;
     }
 
+    public ExtraData getOtherPartyExtraData() {
+        return otherPartyExtraData;
+    }
+
+    public String getError() {
+        return error;
+    }
```

Comment:
Regarding https://github.com/getyoti/yoti-java-sdk/pull/380#discussion_r1129238821 does returning an `Optional` wrapped value not be preferable for all the values that _might_ not exist?